.travis.yml

```diff
@@ -29,6 +29,14 @@ branches:
   except:
     - /^[0-9]/
 
+jobs:
+  include:
+    - script: ./mvnw clean test -Dversion.io.vertx=3.3.0
```

Comment:
I'm not an expert about Travis, but I think that you could use the `matrix` directive, like there is for Jaeger. The advantage is that it's easier to see which element it is about: https://travis-ci.org/uber/jaeger/builds/269000111

Example: https://github.com/jpkrohling/jaeger/blob/master/.travis.yml#L7-L24

Reply:
Maybe I am wrong but I think jaeger uses matrix because by that time [stages](https://docs.travis-ci.com/user/build-stages/) were not available. cc @yurishkuro 

By matrix I understand it creates all combinations which is not needed here.